woocommerce/woocommerce-paypal-payments
# Save Card Last Digits in order meta for Advanced Card Payments (2935)

### Description

This PR adds saving of the Last Digits in order meta.

### Steps to Test




1. Create a new order.
2. Pay with ACDC
3. Verify the Last Digits have been added to the order meta (not just order notes).
